migrations/1599523522436-migrate-config.js

```diff
@@ -1,34 +1,35 @@
 const db = require('./db')
-const settingsLoader = require('../lib/admin/settings-loader')
 const machineLoader = require('../lib/machine-loader')
-const { saveConfig, saveAccounts } = require('../lib/new-settings-loader')
+const { saveConfig, saveAccounts, loadLatest } = require('../lib/new-settings-loader')
 const { migrate } = require('../lib/config-migration')
 
+const OLD_SETTINGS_LOADER_SCHEMA_VERSION = 1
+
 module.exports.up = function (next) {
-  function migrateConfig(settings) {
+  function migrateConfig (settings) {
     const newSettings = migrate(settings.config, settings.accounts)
     return Promise.all([
       saveConfig(newSettings.config),
       saveAccounts(newSettings.accounts)
     ])
-    .then(() => next())
+      .then(() => next())
   }
 
-  settingsLoader.loadLatest(false)
+  loadLatest(OLD_SETTINGS_LOADER_SCHEMA_VERSION)
     .then(async settings => ({
       settings,
       machines: await machineLoader.getMachineNames(settings.config)
     }))
```

Comment:
this .then mixed with await makes the code seem kinda weird. Can't we change the function to async and use Promise.all() to await all promises here?

something like

```JS
module.exports.up = async function (next) {
	// ...
  const settingsPromise = loadLatest(OLD_SETTINGS_LOADER_SCHEMA_VERSION)
  const machinesPromise = machineLoader.getMachineNames(settings.config)
  try {
  	const [settings, machines] = await Promise.all([settingsPromise, machinesPromise])
    // rest of code
  } catch (err) {
  	// ...
  }
}
```